.gitignore

```diff
@@ -1,4 +1,4 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
-/doc/_build/
+/doc/build/
```

Comment:
@abhishalya , Documenter uses different output folder and .gitignore should be changed. Sorry for not catching that during review.

Reply:
Ah, right. Sorry I missed that too. Thanks for correcting that :)